raiden/network/transport/matrix/rtc/web_rtc.py

```diff
@@ -620,15 +604,17 @@ def stop(self) -> None:
         self._reset_state()
         self._send_hangup_messages()
 
+    def _send_signaling_message(self, address: Address, message: str) -> None:
+        metadata = _query_metadata(self._pfs_config, address)
```

Comment:
Perhaps not as it is, but the idea is to add metadata caching for the duration of the signaling process, and then it will be nicer to have this factored out.